app/components/search/SearchMap/SearchMap.tsx

```diff
@@ -13,14 +13,18 @@ import {
 import "./SearchMap.scss";
 import { TransformedSearchHit } from "../../../models";
 import config from "../../../config";
+import { SearchMapActions } from "components/search/SearchResults/SearchResults";
```

Comment:
**question (non-blocking):** I usually have a separate types dir in my projects, but we don't have one in this. Curious what your thoughts are on exporting types from a component file vs exporting from a types file

Reply:
I've done it both ways and seen it mixed. I don't tend to think about it in absolutes and am not too concerned about consistency for this since it's easy to find them either way with the _goto_ behavior of IDEs. Generally I tend to prefer more colocation than less because then your brain doesn't have to jump around as much. For splitting out types I prefer if the types file is colocated, rather than an entirely separate directory.

Reply:
@adriencyberspace how about we spike an entry in https://github.com/Exygy/consulting-style/blob/main/Typescript.md to describe some preferred approaches!